defaults/main.yml

```diff
@@ -2,8 +2,17 @@
 # defaults file for ansible-coreos-kubelet
 coreos_kubelet_systemd: /etc/systemd/system
 coreos_kubelet_install_dir: /opt/bin
-kubelet_version: "v1.4.1_coreos.0"
+kubelet_version: "v1.7.0_coreos.0"
```

Comment:
That issue is for GCE. Also, I've only tested v1.7.0 so would like to leave the default to that version.
We can always update this from the base repo as needed.